src/test/java/com/cdancy/bitbucket/rest/features/KeysApiMockTest.java

```diff
@@ -0,0 +1,400 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cdancy.bitbucket.rest.features;
+
+import com.cdancy.bitbucket.rest.options.CreateAccessKey;
+import com.cdancy.bitbucket.rest.options.CreateKey;
+import com.cdancy.bitbucket.rest.BitbucketApi;
+import com.cdancy.bitbucket.rest.BitbucketApiMetadata;
+import com.cdancy.bitbucket.rest.domain.common.RequestStatus;
+import com.cdancy.bitbucket.rest.domain.sshkey.AccessKey.PermissionType;
+import com.cdancy.bitbucket.rest.domain.sshkey.AccessKeyPage;
+import com.cdancy.bitbucket.rest.domain.sshkey.AccessKey;
+import com.cdancy.bitbucket.rest.BaseBitbucketMockTest;
+import com.google.common.collect.ImmutableMap;
+import com.squareup.okhttp.mockwebserver.MockResponse;
+import com.squareup.okhttp.mockwebserver.MockWebServer;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Mock tests for the {@link KeysApi} class.
+ */
+@Test(groups = "unit", testName = "KeysApiMockTest")
+public class KeysApiMockTest extends BaseBitbucketMockTest {
+
+    private final String projectKey = "MY-PROJECT";
+    private final String repoKey = "my-repo";
+    private final int keyId = 1;
+    private final String getMethod = "GET";
+    private final String deleteMethod = "DELETE";
```

Comment:
:warning: This final field could be made static (#QR195:48#)-PMD_FinalFieldCouldBeStatic